code/game/objects/items/halloweendecorations.dm

```diff
@@ -1,26 +1,4 @@
 //Two small halloween decoration items
-//Candle pile
```

Comment:
you can keep it here, just make it a child and remove the duplicate light() proc. you can keep the Initialize() since its unique.